vector/src/main/java/im/vector/app/features/pin/lockscreen/ui/LockScreenFragment.kt

```diff
@@ -164,7 +164,7 @@ class LockScreenFragment : VectorBaseFragment<FragmentLockScreenBinding>() {
 
     private fun setupCodeView(lockScreenCodeView: LockScreenCodeView, configuration: LockScreenConfiguration) = with(lockScreenCodeView) {
         codeLength = configuration.pinCodeLength
-        onCodeCompleted = { code ->
+        onCodeCompleted = LockScreenCodeView.CodeCompletedListener { code ->
```

Comment:
Use fonction reference?

Reply:
Sorry, I'm not following. How should I use a function reference here? By extracting its contents to private function first?

Reply:
Oh, ok. Honestly, I don't think its worth it, we'd just be moving the `viewModel.handle(LockScreenAction.PinCodeEntered(code))` code into a `handleCodeCompleted(code: String)` which probably doesn't add much value. If this was used in several places it could make sense.